RecoEgamma/EgammaPhotonProducers/src/PhotonProducer.cc

```diff
@@ -1,483 +1,505 @@
 
 #include "DataFormats/EgammaReco/interface/ElectronSeed.h"
 #include "RecoCaloTools/Selectors/interface/CaloConeSelector.h"
 
-#include "RecoLocalCalo/EcalRecAlgos/interface/EcalSeverityLevelAlgoRcd.h"
 #include "RecoLocalCalo/EcalRecAlgos/interface/EcalSeverityLevelAlgo.h"
+#include "RecoLocalCalo/EcalRecAlgos/interface/EcalSeverityLevelAlgoRcd.h"
 
-#include "RecoEgamma/EgammaPhotonProducers/interface/PhotonProducer.h"
-#include "RecoEgamma/EgammaIsolationAlgos/interface/EgammaTowerIsolation.h"
 #include "RecoEcal/EgammaCoreTools/interface/EcalClusterFunctionBaseClass.h"
 #include "RecoEcal/EgammaCoreTools/interface/EcalClusterFunctionFactory.h"
 #include "RecoEcal/EgammaCoreTools/plugins/EcalClusterCrackCorrection.h"
 #include "RecoEgamma/EgammaIsolationAlgos/interface/EgammaHadTower.h"
+#include "RecoEgamma/EgammaIsolationAlgos/interface/EgammaTowerIsolation.h"
+#include "RecoEgamma/EgammaPhotonProducers/interface/PhotonProducer.h"
 
-PhotonProducer::PhotonProducer(const edm::ParameterSet& config) :
-
-  conf_(config)
-{
+PhotonProducer::PhotonProducer(const edm::ParameterSet& config)
+    :
 
+      conf_(config) {
   // use onfiguration file to setup input/output collection names
 
-  photonCoreProducer_   =
-    consumes<reco::PhotonCoreCollection>(conf_.getParameter<edm::InputTag>("photonCoreProducer"));
-  barrelEcalHits_   =
-    consumes<EcalRecHitCollection>(conf_.getParameter<edm::InputTag>("barrelEcalHits"));
-  endcapEcalHits_   =
-    consumes<EcalRecHitCollection>(conf_.getParameter<edm::InputTag>("endcapEcalHits"));
-  vertexProducer_   =
-    consumes<reco::VertexCollection>(conf_.getParameter<edm::InputTag>("primaryVertexProducer"));
-  hcalTowers_ =
-    consumes<CaloTowerCollection>(conf_.getParameter<edm::InputTag>("hcalTowers"));
-  hOverEConeSize_   = conf_.getParameter<double>("hOverEConeSize");
-  highEt_        = conf_.getParameter<double>("highEt");
+  photonCoreProducer_ = consumes<reco::PhotonCoreCollection>(
+      conf_.getParameter<edm::InputTag>("photonCoreProducer"));
+  barrelEcalHits_ = consumes<EcalRecHitCollection>(
+      conf_.getParameter<edm::InputTag>("barrelEcalHits"));
+  endcapEcalHits_ = consumes<EcalRecHitCollection>(
+      conf_.getParameter<edm::InputTag>("endcapEcalHits"));
+  vertexProducer_ = consumes<reco::VertexCollection>(
+      conf_.getParameter<edm::InputTag>("primaryVertexProducer"));
+  hcalTowers_ = consumes<CaloTowerCollection>(
+      conf_.getParameter<edm::InputTag>("hcalTowers"));
+  hOverEConeSize_ = conf_.getParameter<double>("hOverEConeSize");
+  highEt_ = conf_.getParameter<double>("highEt");
   // R9 value to decide converted/unconverted
-  minR9Barrel_        = conf_.getParameter<double>("minR9Barrel");
-  minR9Endcap_        = conf_.getParameter<double>("minR9Endcap");
-  usePrimaryVertex_   = conf_.getParameter<bool>("usePrimaryVertex");
-  runMIPTagger_       = conf_.getParameter<bool>("runMIPTagger");
+  minR9Barrel_ = conf_.getParameter<double>("minR9Barrel");
```

Comment:
was the previous "pretty-format" modified by some clang-pretty script or something?
or was it some manual untabify or something similar?
the old piece was more readable